src/datasource_toolkit/forestadmin/datasource_toolkit/decorators/search/collections.py

```diff
@@ -124,10 +127,19 @@ def _get_searchable_fields(self, collection: Collection, extended: bool) -> List
             if is_column(field):
                 fields.append((name, field))
 
-            if extended and (is_many_to_one(field) or is_one_to_one(field)):
+            if extended and (is_many_to_one(field) or is_one_to_one(field) or is_polymorphic_one_to_one(field)):
                 related = collection.datasource.get_collection(field["foreign_collection"])
 
                 for sub_name, sub_field in related.schema["fields"].items():
                     if is_column(sub_field):
                         fields.append((f"{name}:{sub_name}", sub_field))
+
+            if extended and is_polymorphic_many_to_one(field):
+                ForestLogger.log(
+                    "info",
+                    f"We're not searching through {self.name}.{name} because it's a polymorphic relation. "
+                    "You can override the default search behavior with 'replace_search'."
+                    " See more: https://docs.forestadmin.com/developer-guide-agents-python/agent-customization/search",
```

Comment:
useless comma after "python/agent-customization/search" right ?

Reply:
yes you are right, but notice it's not in the string, it's a parameter separator. 
So do your job and let my code formatter do his.